aarch64 build fails

<!--
Please verify first that your issue is not already reported on GitHub.

This issue tracker is only for application and packaging bugs. Other kinds of
bugs must be reported at other locations:
 - Documentation: https://github.com/LibrePCB/librepcb-doc
 - Translations: https://www.transifex.com/librepcb/librepcb-application/
 - Website: https://github.com/LibrePCB/LibrePCB.github.io
-->

##### VERSION / OS / ENVIRONMENT
<!-- Paste the exact application version, operating system and environment used
below. You can copy all needed information directly from the "Help->About"
dialog, tab "Details". Alternatively you could copy the version numbers from the
console output (the first few lines). -->

```
LibrePCB Version: 1.0.0-rc1
Git Revision: 2d9ea3c230ef8aa5c431216a3daac829b56a617c
Qt Version: 5.15
Operating System: Flatpak runtime org.kde.Platform//5.15-22.08
```

##### SUMMARY
The aarch64 build of version 1.0.0-rc1 fails while the x86_64 build succeeds.


##### STEPS TO REPRODUCE
Run `flatpak-builder --force-clean --ccache --repo=librepcb-repo librepcb-build org.librepcb.LibrePCB` with the following manifest on an aarch64 machine or use a Flathub buildbot:
```
{
    "app-id": "org.librepcb.LibrePCB",
    "runtime": "org.kde.Platform",
    "runtime-version": "5.15-22.08",
    "sdk": "org.kde.Sdk",
    "command": "librepcb",
    "finish-args": [
        "--device=dri",
        "--socket=x11",
        "--socket=wayland",
        "--share=ipc",
        "--share=network", /* required for downloading libraries */
        "--filesystem=home" /* required for storing workspace, libraries etc. */
    ],
    "modules": [
        "shared-modules/glu/glu-9.json",
        {
            "name": "librepcb",
            "buildsystem": "cmake-ninja",
            "builddir": true,
            "config-opts": [
                "-DUSE_OPENCASCADE=0"
            ],
            "sources": [
                {
                    "type": "git",
                    "url": "https://github.com/LibrePCB/LibrePCB",
                    "tag": "1.0.0-rc1",
                    "commit": "2d9ea3c230ef8aa5c431216a3daac829b56a617c"
                }
            ]
        }
    ]
}
```


##### EXPECTED RESULTS
I expect the build to succeed.

`[681/833] Building CXX object libs/librepcb/editor/CMakeFiles/librepcb_editor.dir/widgets/openglview.cpp.o`


##### ACTUAL RESULTS
```
[660/833] Building CXX object libs/librepcb/editor/CMakeFiles/librepcb_editor.dir/widgets/openglview.cpp.o
FAILED: libs/librepcb/editor/CMakeFiles/librepcb_editor.dir/widgets/openglview.cpp.o 
/usr/bin/c++ -DQT_CONCURRENT_LIB -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_PRINTSUPPORT_LIB -DQT_QMLMODELS_LIB -DQT_QML_LIB -DQT_QUICKCONTROLS2_LIB -DQT_QUICK_LIB -DQT_SQL_LIB -DQT_SVG_LIB -DQT_WIDGETS_LIB -DTYPE_SAFE_ARITHMETIC_POLICY=1 -DTYPE_SAFE_ENABLE_ASSERTIONS=0 -DTYPE_SAFE_ENABLE_PRECONDITION_CHECKS=1 -DTYPE_SAFE_ENABLE_WRAPPER=1 -I/run/build/librepcb/_flatpak_build/libs/librepcb/editor -I/run/build/librepcb/libs/librepcb/editor -I/run/build/librepcb/_flatpak_build/libs/librepcb/editor/librepcb_editor_autogen/include -I/run/build/librepcb/libs/librepcb/editor/../.. -I/run/build/librepcb/_flatpak_build/libs/librepcb/core -I/run/build/librepcb/libs/optional/tl -I/run/build/librepcb/libs/type_safe/include -I/run/build/librepcb/libs/type_safe/external/debug_assert -isystem /usr/include/QtConcurrent -isystem /usr/include/QtCore -isystem /usr/lib/mkspecs/linux-g++ -isystem /usr/include/QtGui -isystem /usr/include/QtNetwork -isystem /usr/include/QtPrintSupport -isystem /usr/include/QtWidgets -isystem /usr/include/QtSql -isystem /usr/include/QtSvg -isystem /usr/include/QtOpenGL -isystem /usr/include/QtQuick -isystem /usr/include/QtQmlModels -isystem /usr/include/QtQml -isystem /usr/include/QtQuickControls2 -O2 -g -pipe -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -fasynchronous-unwind-tables -fstack-clash-protection -fno-omit-frame-pointer -O3 -DNDEBUG -std=c++11 -Wall -Wextra -Wpedantic -Wno-noexcept-type -Wno-deprecated-declarations -fPIC -MD -MT libs/librepcb/editor/CMakeFiles/librepcb_editor.dir/widgets/openglview.cpp.o -MF libs/librepcb/editor/CMakeFiles/librepcb_editor.dir/widgets/openglview.cpp.o.d -o libs/librepcb/editor/CMakeFiles/librepcb_editor.dir/widgets/openglview.cpp.o -c /run/build/librepcb/libs/librepcb/editor/widgets/openglview.cpp
/run/build/librepcb/libs/librepcb/editor/widgets/openglview.cpp: In member function ‘virtual void librepcb::editor::OpenGlView::initializeGL()’:
/run/build/librepcb/libs/librepcb/editor/widgets/openglview.cpp:242:12: error: ‘GL_MULTISAMPLE’ was not declared in this scope; did you mean ‘GL_MULTISAMPLES_NV’?
  242 |   glEnable(GL_MULTISAMPLE);
      |            ^~~~~~~~~~~~~~
      |            GL_MULTISAMPLES_NV
/run/build/librepcb/libs/librepcb/editor/widgets/openglview.cpp:245:12: error: ‘GL_LINE_SMOOTH’ was not declared in this scope; did you mean ‘GL_LINE_LOOP’?
  245 |   glEnable(GL_LINE_SMOOTH);
      |            ^~~~~~~~~~~~~~
      |            GL_LINE_LOOP
/run/build/librepcb/libs/librepcb/editor/widgets/openglview.cpp:246:10: error: ‘GL_LINE_SMOOTH_HINT’ was not declared in this scope
  246 |   glHint(GL_LINE_SMOOTH_HINT, GL_NICEST);
      |          ^~~~~~~~~~~~~~~~~~~
```


##### ADDITIONAL INFORMATION
<!-- Optionally you can provide additional information, for example:
 - Console output / stack trace (if large, only link to gist.github.com or so)
 - Link to your library or project files
 - Screenshots
-->

--- a/libs/librepcb/editor/widgets/openglview.cpp
+++ b/libs/librepcb/editor/widgets/openglview.cpp
@@ -29,6 +29,18 @@
 #include <QtOpenGL>
 #include <QtWidgets>
 
+// Compatibility defines to fix build error on some targets, see
+// https://github.com/LibrePCB/LibrePCB/issues/1205.
+#ifndef GL_MULTISAMPLE
+#define GL_MULTISAMPLE 0x809D
+#endif
+#ifndef GL_LINE_SMOOTH
+#define GL_LINE_SMOOTH 0x0B20
+#endif
+#ifndef GL_LINE_SMOOTH_HINT
+#define GL_LINE_SMOOTH_HINT 0x0C52
+#endif
+
 /*******************************************************************************
  *  Namespace
  ******************************************************************************/

